ui/src/app/edge/live/thermometer/thermometer.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { ChannelAddress, CurrentData, EdgeConfig, Utils } from 'src/app/shared/shared';
+import { AbstractFlatWidget } from 'src/app/shared/genericComponents/flat/abstract-flat-widget';
+import { formatNumber } from '@angular/common';
+
+@Component({
+  selector: 'Thermometer',
+  templateUrl: './thermometer.component.html'
+})
+export class ThermometerComponent extends AbstractFlatWidget {
+
+  public sensors: EdgeConfig.Component[] = null;
+  public readonly CONVERT_TO_Degree = Utils.CONVERT_TO_Degree;
```

Comment:
can you change this Callback into fully Uppercase? This is how we did it for the other Converters as well